src/openzaak/components/catalogi/tests/test_filters.py

```diff
@@ -0,0 +1,237 @@
+from collections import OrderedDict
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+from vng_api_common.tests import get_validation_errors, reverse
+
+from openzaak.utils.tests import JWTAuthMixin
+
+from ..models import (
+    BesluitType,
+    Eigenschap,
+    InformatieObjectType,
+    ResultaatType,
+    RolType,
+    StatusType,
+    ZaakInformatieobjectType,
+    ZaakType,
+)
+
+
+class BesluitTypeFilterTests(JWTAuthMixin, APITestCase):
+    heeft_alle_autorisaties = True
+
+    def test_filter_by_invalid_url(self):
+        for query_param in ["catalogus", "zaaktypes", "informatieobjecttypes"]:
+            with self.subTest(query_param=query_param):
+                response = self.client.get(reverse(BesluitType), {query_param: "bla"})
+
+                self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+                error = get_validation_errors(response, query_param)
+                self.assertEqual(error["code"], "invalid")
+
+    def test_filter_by_valid_url_object_does_not_exist(self):
+        for query_param in ["catalogus", "zaaktypes", "informatieobjecttypes"]:
+            with self.subTest(query_param=query_param):
```

Comment:
can you please create `Besluittype` object with no m2m relations to zaaktypes and informatieobjecttypes before making a request?

Reply:
could you please add `concept=False` when creating Besluittype? This can affect the result of the request

Reply:
I have added `concept=False` (https://github.com/open-zaak/open-zaak/pull/133/commits/22337ffa7ffe86af9dcaa7fb6498ea0bd106dde5) where needed and the tests did indeed fail after that, so I fixed the issue in https://github.com/VNG-Realisatie/vng-api-common/pull/107

I also added factories to the other filter tests and that made them fail as well, so I fixed that too in the vng_api_common PR

Thanks for the feedback